File: C_C++_Dev/POSN 57/2_Problem/guess/grader.pas
program grader;
   uses gradershare, player;
   var r: longint;
begin
   read(N,secret);
   r := play(N);
   if r = secret then
      Writeln ('Correct ',r)
   else
      Writeln ('Incorrect ',r,' should be ',secret);
end.

File: C_C++_Dev/POSN 57/2_Problem/guess/grader.c
#include "player.c"
#include "grader.h"
#include <stdio.h>

static int N,J,g,n;

int bigger(int K) {
   return J > K;
}

int main(){
   scanf("%d%d",&N,&J);
   g = play(N);
   if (g == J) {
      printf("Correct %d\n",J);
   } else {
      printf("Incorrect %d should be %d\n",g,J);
   }
}
